RecoEcal/EgammaClusterProducers/test/BuildFile.xml

```diff
@@ -8,3 +8,35 @@
   <use name="FWCore/ParameterSet"/>
   <flags EDM_PLUGIN="1"/>
 </library>
+
+<library name="DRNTestNTuplizer" file="DRNTestNTuplizer.cc">
```

Comment:
a lot of new dependencies... Does it indeed need to be in cms-sw/cmssw master, or is this private code?

Reply:
Any news on this?